independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenNEM",
+    "description": "An Open Platform for Australian Electricity Market Data",

```

Comment:
@paulculmsee Can you please provide a more detailed description which can be understandable by everyone and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
Okay. I made a note in the certifying step that this is an open source API and they dont publish the schema, so they will be terse. Cool with that?